cmd/tendermint/commands/reset.go

```diff
@@ -13,81 +13,71 @@ import (
 	"github.com/tendermint/tendermint/types"
 )
 
-// MakeResetAllCommand constructs a command that removes the database of
+// MakeResetCommand constructs a command that removes the database of
 // the specified Tendermint core instance.
-func MakeResetAllCommand(conf *config.Config, logger log.Logger) *cobra.Command {
-	var keyType string
+func MakeResetCommand(conf *config.Config, logger log.Logger) *cobra.Command {
+	var (
+		resetAllFlag       bool
+		resetPrivValFlag   bool
+		resetPeerStoreFlag bool
+		keyType            string
+	)
 
 	cmd := &cobra.Command{
-		Use:   "unsafe-reset-all",
-		Short: "(unsafe) Remove all the data and WAL, reset this node's validator to genesis state",
+		Use:   "reset",
+		Short: "Remove all tendermint data, resetting to genesis state.",
+		Long: `Removes block, state and evidence store. Does not alter priv-validator state or your peer store.
+If resetting, don't forget to reset application state as well.`,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return resetAll(conf.DBDir(), conf.PrivValidator.KeyFile(),
-				conf.PrivValidator.StateFile(), logger, keyType)
+			if resetAllFlag {
+				return ResetAll(conf.DBDir(), conf.PrivValidator.KeyFile(),
+					conf.PrivValidator.StateFile(), logger, keyType)
+			}
+			if resetPrivValFlag {
```

Comment:
In cases where we're not resetting all, it might make more sense to apply _all_ the flagged resets. Otherwise it's kind of confusing which one you get if you specify multiple. 

Alternatively we could reject multiple flags.

Reply:
That's true. I'm fine with either. We could also make them subcommands of `reset`

Reply:
Separate subcommands would also work. If it's better to have them be separate (rather than being able to pick multiple) that might be better.

Reply:
I like separate sub-commands.

Reply:
Added separate sub-commands